net/nginx/files/10_nginx-conf_set_worker_processes

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+core_number=$(grep -c ^processor /proc/cpuinfo)
+sed -i "s/^\s*worker_processes\s.*/worker_processes "$core_number";/" \
+	/etc/nginx/nginx.conf 
```

Comment:
One thing.
We have also return with ```exit 0``` after script run.
If not then the script will run on every boot.
It will not be removed after first boot
Have a look into the files in https://github.com/openwrt/openwrt/tree/master/package/base-files/files/etc/uci-defaults

Reply:
I fix that, thanks.